src/screens/home/HomeScreen.tsx

```diff
@@ -1,27 +1,27 @@
-import React, {useMemo, useState, useEffect} from 'react';
-import {AppState, AppStateStatus} from 'react-native';
-import {BottomSheet, Box} from 'components';
-import {useExposureStatus, useSystemStatus, SystemStatus} from 'services/ExposureNotificationService';
-import {checkNotifications, requestNotifications} from 'react-native-permissions';
-import {useNetInfo} from '@react-native-community/netinfo';
+import React, { useMemo, useState, useEffect } from 'react';
+import { AppState, AppStateStatus } from 'react-native';
+import { BottomSheet, Box } from 'components';
```

Comment:
👾

Reply:
You might need to download Prettier https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode in VSCode and format your code with `Shift + Option + F`